pkg/keystore/awsKeystore.go

```diff
@@ -57,8 +60,9 @@ func (keystore AwsKeystore) EncryptAndWrite(initResponse *api.InitResponse) erro
 }
 
 func (keystore AwsKeystore) ReadAndDecrypt() (*api.InitResponse, error) {
+	secretPath := filepath.Join(keystore.secretsPath, unsealKeysFile)
```

Comment:
I wouldn't use `filepath` if I where you. The implementation of it depends on the current `GOOS` to figure out how to handle paths. Which is not what you want in this situation. Concatenate the strings instead.